main/com.spotify/index.js

```diff
@@ -60,6 +61,10 @@ module.exports = class SpotifyDevice extends Tp.BaseDevice {
         this._state = new Map();
         this._queryResults = new Map();
         this._deviceState = new Map();
+
+        if (this.platform.type == "server") {
+            this.spotifyd = new spotifyd({ cacheDir: this.platform._cacheDir, username: this.state.id, device_name: this.state.id, token: this.accessToken });
+        }
```

Comment:
Minor: our code style says no braces if the body has one line (unless the other branch needs braces). eslint should flag this.
Also code style: please no tabs. 4 spaces everywhere.